x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/warm_phase/warm_phase.js

```diff
@@ -195,6 +195,12 @@ class WarmPhaseUi extends PureComponent {
                         errorKey={PHASE_REPLICA_COUNT}
                         isShowingErrors={isShowingErrors}
                         errors={errors}
+                        helpText={
```

Comment:
Looks like the containing element has a max-width set on it, causing the text to wrap. You could try removing it and applying it to `EuiFieldNumber` on ln 205 instead.

![image](https://user-images.githubusercontent.com/1238659/49974397-d63fa680-feed-11e8-9f8b-9edf8cd7e35c.png)